conf/broker.conf

```diff
@@ -283,6 +283,24 @@ athenzDomainNames=
 # When this parameter is not empty, unauthenticated users perform as anonymousUserRole
 anonymousUserRole=
 
+### --- Token Authentication Provider --- ###
```

Comment:
need same in proxy.conf also.

Reply:
Also, site2/docs/reference-configuration.md (for both broker and proxy)

Reply:
Fixed. Though, we'd really should automate this. It's already wildly out of date.

Reply:
+1000 for automation